### PR TITLE
CNDIT-1517: Increase test coverage to 90% and more

### PR DESCRIPTION
**Pull request by sveselev**
## Description

Increased the test coverage for RTR post-processing-service to 94%. Rolled back the changes for notification ids processing in the context of upcoming pre-processing re-factoring.

- **PostProcRepository.java**: combined JPA repository for all stored procedures that currently do not return data
- **PostProcSp.java**: root fake data entity containing sp result
- **PostProcessingService.java**: reverted notification processing, adapted to common PostProcRepository
- **PostProcessingServiceTest.java**: added several edge case tests
- **024-sp_nrt_notification_postprocessing-001.sql**: reverted sp script to process notification uids

## JIRA
[CNDIT-1517](https://cdc-nbs.atlassian.net/browse/CNDIT-1517)

### CODE REVIEW

File: liquibase-service/src/main/resources/db/rdb/024-sp_nrt_notification_postprocessing-001.sql [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CREATE OR ALTER PROCEDURE [dbo].[sp_nrt_notification_postprocessing] @phc_uids nvarchar(max), @debug bit = 'false'
+CREATE   PROCEDURE [dbo].[sp_nrt_notification_postprocessing] @notification_uids nvarchar(max), @debug bit = 'false'
```
Comment (upasanapattnaik-eq) [inline liquibase-service/src/main/resources/db/rdb/024-sp_nrt_notification_postprocessing-001.sql:1]: I believe we need to add the "OR ALTER" for Liquibase here

**Review by upasanapattnaik-eq — APPROVED** [review]
LGTM! Thank you so much for the updates.

**Review by alog-enquizit — APPROVED** [review]
